examples/low_level_components.py

```diff
@@ -0,0 +1,115 @@
+"""
+An example on how to send and process components without using views.
+"""
+
+import disnake
+from disnake.ext import commands
+
+bot = commands.Bot(command_prefix=commands.when_mentioned)
+
+
+# As an alternative to using views, it is possible to use a more low-level approach to components.
+# Firstly, components do not have to be sent as part of a view. Instead, they can be sent as-is.
+# Take special note of the fact that `custom_id`s have been explicitly set on these components.
+
+# The main advantage of this is that listeners are, by nature, persistent. Listeners also do not
+# suffer views' problem of causing a memory leak, where view instances will keep piling up.
```

Comment:
tbh we shouldn't mention that views have a memory leak, as that's an implementation detail, and we are working on fixing this issue.

Reply:
Or just note that it's a leftover from discord.py and blame it on them. 👍

Reply:
The increased memory usage of views should be mentioned, but yeah, the fact that views are kept in memory longer than necessary is something that is planned to be fixed.

Reply:
Alright, changed it ^^